src/order/filter/lift.lean

```diff
@@ -360,9 +366,13 @@ theorem comap_eq_lift' {f : filter β} {m : α → β} :
   comap m f = f.lift' (preimage m) :=
 filter.ext $ λ s, (mem_lift'_sets monotone_preimage).symm
 
-lemma lift'_infi_powerset [nonempty ι] {f : ι → filter α} :
+lemma lift'_infi_powerset {f : ι → filter α} :
   (infi f).lift' powerset = (⨅i, (f i).lift' powerset) :=
-lift'_infi $ λ _ _, (powerset_inter _ _).symm
+begin
+  by_cases hι : nonempty ι,
```

Comment:
I'm curious whether the `nontriviality` tactic is useful here, but the current proof is probably faster anyway

Reply:
I don't think that `simp` will prove this goal assuming `[subsingleton ι]`.